tests/python_client/common/common_func.py

```diff
@@ -87,7 +91,14 @@ def gen_binary_vec_field(name=ct.default_binary_vec_field_name, is_primary=False
 
 def gen_default_collection_schema(description=ct.default_desc, primary_field=ct.default_int64_field_name,
                                   auto_id=False, dim=ct.default_dim):
-    fields = [gen_int64_field(), gen_float_field(), gen_float_vec_field(dim=dim)]
+    fields = [gen_int64_field(), gen_float_field(), gen_string_field(), gen_float_vec_field(dim=dim)]
+    schema, _ = ApiCollectionSchemaWrapper().init_collection_schema(fields=fields, description=description,
+                                                                    primary_field=primary_field, auto_id=auto_id)
+    return schema
+
+def gen_default_collection_string_schema(description=ct.default_desc, primary_field=ct.default_string_field_name,
+                                  auto_id=False, dim=ct.default_dim):
+    fields = [gen_int64_field(), gen_float_field(), gen_string_field(is_primary=True), gen_float_vec_field(dim=dim)]
```

Comment:
the hard code of "is_primay=True" could be conflict with primary_field in next line of schema init,  if primary_field is passed with int or something else.

Reply:
Modified, please review